solr/contrib/ltr/README.md

```diff
@@ -1,390 +1,24 @@
-or more of your features, and text1 is the information to be pass in. As an
-example that matches the earlier shown userTextTitleMatch feature one could do:
-
-`rq={!ltr reRankDocs=3 model=externalmodel efi.user_text='Casablanca' efi.user_intent='movie'}`
-
-# Extract features
-To extract features you need to use the feature vector transformer `features`
-
-`fl=*,score,[features]&rq={!ltr model=yourModel reRankDocs=25}`
-
-If you use `[features]` together with your reranking model, it will return
-the array of features used by your model. Otherwise you can just ask solr to
-produce the features without doing the reranking:
-
-`fl=*,score,[features store=yourFeatureStore format=[dense|sparse] ]`
-
-This will return the values of the features in the given store. The format of the 
-extracted features will be based on the format parameter. The default is dense.
-
-# Assemble training data
-In order to train a learning to rank model you need training data. Training data is
-what "teaches" the model what the appropriate weight for each feature is. In general
-training data is a collection of queries with associated documents and what their ranking/score
-should be. As an example:
-```
-secretary of state|John Kerry|0.66|CROWDSOURCE
-secretary of state|Cesar A. Perales|0.33|CROWDSOURCE
-secretary of state|New York State|0.0|CROWDSOURCE
-secretary of state|Colorado State University Secretary|0.0|CROWDSOURCE
-
-microsoft ceo|Satya Nadella|1.0|CLICK_LOG
-microsoft ceo|Microsoft|0.0|CLICK_LOG
-microsoft ceo|State|0.0|CLICK_LOG
-microsoft ceo|Secretary|0.0|CLICK_LOG
-```
-In this example the first column indicates the query, the second column indicates a unique id for that doc,
-the third column indicates the relative importance or relevance of that doc, and the fourth column indicates the source.
-There are 2 primary ways you might collect data for use with your machine learning algorithim. The first
-is to collect the clicks of your users given a specific query. There are many ways of preparing this data
-to train a model (http://www.cs.cornell.edu/people/tj/publications/joachims_etal_05a.pdf). The general idea
-is that if a user sees multiple documents and clicks the one lower down, that document should be scored higher
-than the one above it. The second way is explicitly through a crowdsourcing platform like Mechanical Turk or
-CrowdFlower. These platforms allow you to show human workers documents associated with a query and have them
-tell you what the correct ranking should be.
+# Getting Started With Solr Learning To Rank
 
-At this point you'll need to collect feature vectors for each query document pair. You can use the information
-from the Extract features section above to do this. An example script has been included in example/train_and_upload_demo_model.py.
+For information on how to get started with solr ltr please see:
+[Solr Reference Guide's section on Result Reranking](https://cwiki.apache.org/confluence/display/solr/Result+Reranking)
```

Comment:
Had considered that but decided to not directly link to the 'example readme' since the example folder would (I think) not be part of the official distribution (whereas the readme itself would be) and for anyone looking at the content on github.com it would be fairly obvious that there is a 'example' sub-directory.

Having said all that, the plan is for the Solr Ref Guide to link to the example directory (on github).